package-lock.json

```diff
@@ -2963,12 +2963,6 @@
             "node-releases": "^1.1.66"
           }
         },
-        "caniuse-lite": {
```

Comment:
I'm surprised to see caniuse bumped automatically like this...not sure what triggered that?

Reply:
I think I might have run some sort of npm update or something